### PR TITLE
BJEJ-48: Replaced static candidates on vote page for dynamic ones

### PR DESCRIPTION
**Pull request by Elrhino**

### CODE REVIEW

File: src/main/java/com/arcbees/bourseje/client/application/vote/CandidateVoteWidget.java [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+
+    public CandidateVoteWidget(Candidate candidate) {
+        widget = binder.createAndBindUi(this);
+
+        inputRadio.setId(candidateId);
+
+        company.setInnerText(candidate.getCompany());
+        name.setInnerText(candidate.getName());
+
+        if(candidate.getPicture() != null) {
```
Comment (meriouma) [inline src/main/java/com/arcbees/bourseje/client/application/vote/CandidateVoteWidget.java]: missing space

**Comment by meriouma** [comment]
LGTM

File: src/main/java/com/arcbees/bourseje/client/application/vote/CandidateVoteWidget.java [hunk 2]
```diff
@@ -0,0 +1,51 @@
+package com.arcbees.bourseje.client.application.vote;
```
Comment (olafleur) [inline src/main/java/com/arcbees/bourseje/client/application/vote/CandidateVoteWidget.java]: copyright

File: src/main/java/com/arcbees/bourseje/client/application/vote/VotePresenter.java [hunk 3]
```diff
@@ -31,8 +35,11 @@
 import com.gwtplatform.mvp.client.proxy.ProxyPlace;
 import com.gwtplatform.mvp.shared.proxy.PlaceRequest;
 
+import java.util.List;
```
Comment (olafleur) [inline src/main/java/com/arcbees/bourseje/client/application/vote/VotePresenter.java]: order

File: src/main/java/com/arcbees/bourseje/client/application/vote/VoteView.java [hunk 4]
```diff
@@ -27,6 +27,9 @@
 import com.google.gwt.user.client.ui.Widget;
 import com.gwtplatform.mvp.client.ViewWithUiHandlers;
 
+import javax.inject.Inject;
```
Comment (olafleur) [inline src/main/java/com/arcbees/bourseje/client/application/vote/VoteView.java]: order

**Pull request merged**